nouvelles-perspectives-des-sciences-sociales.csl

```diff
@@ -0,0 +1,606 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="text" initialize="false" demote-non-dropping-particle="never" default-locale="fr-CA">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Nouvelles Perspectives des Sciences Sociales (French - Canada)</title>
+    <title-short>NPSS</title-short>
+    <id>http://www.zotero.org/styles/nouvelles-perspectives-des-sciences-sociales</id>
+    <link href="http://www.zotero.org/styles/nouvelles-perspectives-des-sciences-sociales" rel="self"/>
+    <link href="http://www.zotero.org/styles/npssrevue" rel="template"/>
+    <link href="http://npssrevue.ca/guide/biblio/" rel="documentation"/>
+    <author>
+      <name>Ines Bouguerra</name>
+      <email>IBouguerra@laurentienne.ca</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <summary>Ce style est basé sur les règles décrites dans le document « Bibliographie et notes : consignes générales » du site web de la revue NPSS (août 2020).</summary>
+    <updated>2020-10-08T00:48:54+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor" form="short">dir.</term>
+      <term name="editor" form="verb-short">éd. par</term>
+      <term name="translator" form="verb-short">trad. par</term>
+      <term name="translator" form="short">traduction</term>
+      <term name="interviewer" form="verb">entretien réalisé par</term>
+      <term name="in">dans </term>
+      <term name="edition">édition</term>
+      <term name="accessed">consulté le</term>
+      <term name="at">disponible sur :</term>
+      <term name="et-al">et al.</term>
+      <term name="ibid" form="short">ibid</term>
+      <term name="issue" form="short">nᵒ</term>
+      <term name="collection-number" form="short">nᵒ</term>
+      <term name="chapter-number" form="short">chap.</term>
+    </terms>
+  </locale>
+  <macro name="contributors">
+    <choose>
+      <if type="song" match="all" variable="composer">
+        <names variable="composer">
+          <name and="text" delimiter-precedes-last="never" initialize="false" name-as-sort-order="all"/>
+        </names>
+      </if>
+      <else>
+        <names variable="author">
+          <name and="text" delimiter-precedes-last="never" initialize="false" name-as-sort-order="all">
+            <name-part name="family"/>
```

Comment:
what's your goal in specifying the name-part here? The way you have it here, without any styling (like `text-case` of `font-style`) it doesn't do anything.  (Same for all further instances below)

Reply:
We are Following the bibliography of the journal http://npssrevue.ca/en/

Reply:
For wich line ?